packages/tessellate-fragment/config.yaml

```diff
@@ -1,4 +1,7 @@
 APP_PORT: 3002
 MORGAN_FORMAT: dev  # Format preset for request logs.
 MORGAN_THRESHOLD: 0 # Do not log requests with a status code less than this value.
-BUNDLE_SOURCE: http://localhost:3001
+
+sources:
```

Comment:
I'm not sure if mixing upper- and lowercase keys is good. The uppercase keys were chosen because those values are typically overridden with environment variables. "sources" on the other hand is unlikely to be overridden in that way.

Maybe we could extend/wrap nconf later to be not case-sensitive.